src/Library/demos/List Model/main.js

```diff
@@ -0,0 +1,86 @@
+import Gio from "gi://Gio";
+import Gdk from "gi://Gdk";
+import Gtk from "gi://Gtk";
+import Adw from "gi://Adw";
+
+const toggle_list_box = workbench.builder.get_object("toggle_list_box");
+const toggle_flow_box = workbench.builder.get_object("toggle_flow_box");
+const reveal_list_box = workbench.builder.get_object("reveal_list_box");
+const reveal_flow_box = workbench.builder.get_object("reveal_flow_box");
+const list_box = workbench.builder.get_object("list_box");
+const flow_box = workbench.builder.get_object("flow_box");
+const add = workbench.builder.get_object("add");
+const remove = workbench.builder.get_object("remove");
+
+//ListModel initialization and binding
+const model = new Gtk.StringList();
+let item = 1;
+
+function createItemForListBox(listItem) {
+  const listRow = new Adw.ActionRow({
+    title: listItem.string,
+  });
+  return listRow;
+}
+
+function createItemForFlowBox(listItem) {
+  const listRow = new Adw.ActionRow({
+    title: listItem.string,
+  });
+  return listRow;
+}
```

Comment:
I think we should pick a more squarish/grid-like widget for this one, but we can tweak that a bit later.

Reply:
I think what we should do here is return an item like:

```js
new Adw.Bin({
    width_request: 96,
    height_request: 96,
    css_classes: ['card'],
    child: new Gtk.Label({
        label: listItem.string,
        halign: Gtk.Align.CENTER,
        hexpand: true;
        valign: Gtk.Align.CENTER,
    }),
});
```

Then change the `Box` wrapping the `FlowBox` with a `Frame`.